res/README.md

```diff
@@ -17,5 +17,5 @@ Please consult the [license](https://raw.githubusercontent.com/microsoft/cascadi
 
 ### Fonts Included
 
-* Cascadia Code, Cascadia Mono (2004.30)
-   * from microsoft/cascadia-code@5795d16e36748612b726164fd8b59f3c1c8b0788
+* Cascadia Code, Cascadia Mono (2005.15)
+   * from microsoft/cascadia-code@0610f2df4356200adb93cb5bca2221b92ad6ee7e

```

Comment:
I'm not sure what happened, but somehow this escaped the spell checker and resulted in `adb` being flagged

Reply:
So. I looked into this. GitHub did think about running the check:
https://github.com/microsoft/terminal/actions/runs/106064187

❌  Check failure
#### GitHub Actions / Spell checking

#### Error
> An unexpected error occurred when executing this workflow. Please submit a new support request using our Support website:

> https://support.github.com/contact

> Please include this unique ID in your request: 5ACD:14F3:6FEB6:7ACF4:5EBEFF32

https://www.githubstatus.com/incidents/44dmc0tqynrq
https://www.githubstatus.com/incidents/3q541nk49qmw

I'm not sure what this is. While I like GitHub, I haven't found their contact form to be particularly useful, @DHowett-MSFT: do you want to try to contact them and reference the `unique ID`?

(Maybe I should find someone who can talk to me about Azure Pipelines...)